beacon-chain/sync/subscriber_beacon_blocks.go

```diff
@@ -62,6 +63,9 @@ func (s *Service) beaconBlockSubscriber(ctx context.Context, msg proto.Message)
 // This function reconstructs the blob sidecars from the EL using the block's KZG commitments,
 // broadcasts the reconstructed blobs over P2P, and saves them into the blob storage.
 func (s *Service) reconstructAndBroadcastBlobs(ctx context.Context, block interfaces.ReadOnlySignedBeaconBlock) {
+	if block.Version() < version.Deneb {
```

Comment:
temporary, it's addressed in terence's PR https://github.com/prysmaticlabs/prysm/pull/14605